README.md

```diff
@@ -23,7 +23,7 @@ Wreck.get('https://google.com/', function (err, res, payload) {
 var Wreck = require('wreck');
 
 var method = 'GET'; // GET, POST, PUT, DELETE
-var uri    = 'https://google.com/';
+var uri    = '/';
```

Comment:
two reasons:
- consolidate documentation for `options` at one location - [request method](https://github.com/hapijs/wreck#requestmethod-uri-options-callback)
- [basic example](https://github.com/hapijs/wreck#basic) already covers fully qualified url usage. I thought of covering `baseUrl` usage in [advanced example](https://github.com/hapijs/wreck#advanced)

wdyt?